src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String res = "Report for period " + dateFrom + " - " + dateTo + System.lineSeparator();
+        StringBuilder builder = new StringBuilder(res);
+        SimpleDateFormat format = new SimpleDateFormat();
+        format.applyPattern("dd.MM.yyyy");
+        Date fromDate = null;
+        try {
+            fromDate = format.parse(dateFrom);
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        Date toDate = null;
+
+        try {
+            toDate = format.parse(dateTo);
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
```

Comment:
it is bad practice to use `e.printStackTrace();` - better throw RuntimeException in such cases